lib/internal/loader/ModuleRequest.js

```diff
@@ -36,21 +38,31 @@ loaders.set('esm', async (url) => {
 
 // Strategy for loading a node-style CommonJS module
 loaders.set('cjs', async (url) => {
-  return createDynamicModule(['default'], url, (reflect) => {
-    debug(`Loading CJSModule ${url}`);
-    const CJSModule = require('module');
-    const pathname = internalURLModule.getPathFromURL(new URL(url));
-    CJSModule._load(pathname);
+  debug(`Loading CJSModule ${url}`);
+  const CJSModule = require('module');
+  const pathname = internalURLModule.getPathFromURL(new URL(url));
+  const exports = CJSModule._load(pathname);
+  const es = Boolean(exports[esModuleSymbol] || exports.__esModule);
+  const keys = es ? Object.keys(exports) : ['default'];
```

Comment:
Would `Object.getOwnPropertyNames(exports)` make more sense?

Reply:
i chose to use Object.keys so that it only exports enumerable properties. (it says so in the esm doc)